modules/sony/display/cbx.rb

```diff
@@ -0,0 +1,61 @@
+module Sony; end
+module Sony::Display; end
+
+# Documentation: https://docs.google.com/spreadsheets/d/1F8RyaDqLYlKBT7fHJ_PT7yHwH3vVMZue4zUc5McJVYM/edit?usp=sharing
+
+class Sony::Display::CBX
+    include ::Orchestrator::Constants
+    include ::Orchestrator::Transcoder
+
+    # Discovery Information
+    tcp_port 4999
+    descriptive_name 'Sony CBX Display'
+    generic_name :Display
+
+    def on_load
+        on_update
+    end
+
+    def on_update
+    end
+
+    def connected
+        schedule.every('60s') { power? }
+    end
+
+    def disconnected
+        schedule.clear
+    end
+
+    #
+    # Power commands
+    #
+    def power(state, opt = nil)
+        if is_affirmative?(state)
+            send("8C 00 00 02 01 8F", hex_string: true)
+            logger.debug "-- sony display requested to power on"
+        else
+            send("8C 00 00 02 01 8F", hex_string: true)
```

Comment:
This is the same hex string as the power on command. Is this a toggle?

Reply:
We believe so, not 100% sure, hence added the else statement just to be sure.

Reply:
The protocol doc in the comment at the top of this file makes reference to both power on and power off commands?

Regardless, if only a toggle is supported, provide a `power` method that accepts a state param, but always toggles is a little disingenuous. With this user would expect to be able to call `power :on` or `power :off` and have the device state reflect this. If there are no discrete power commands it's worth using the current known state (if this is accurate) to only execute the toggle is required.